src/main/java/io/github/thebusybiscuit/slimefun4/core/services/github/GitHubIssuesTracker.java

```diff
@@ -25,15 +29,16 @@
     }
 
     @Override
-    public void onSuccess(JsonElement element) {
-        if (element.isJsonArray()) {
-            JsonArray array = element.getAsJsonArray();
+    public void onSuccess(JsonNode element) {
+        if (element.isArray()) {
+            JSONArray array = element.getArray();
 
             int issues = 0;
             int pullRequests = 0;
 
-            for (JsonElement elem : array) {
-                JsonObject obj = elem.getAsJsonObject();
+            // Don't even question why this is an Iterable<Object>
+            for (Object elem : array) {
+                JSONObject obj = (JSONObject) elem;
```

Comment:
ewwww, Iterable<Object>, I think you should rather do it like you did in the other class:
https://github.com/TheBusyBiscuit/Slimefun4/pull/2107/files#diff-3e7b2257540224b37738ee5d7874df07R89-R90